services/payments/src/simcore_service_payments/core/settings.py

```diff
@@ -98,6 +106,11 @@ class ApplicationSettings(_BaseApplicationSettings):
         description="Based on this variable is the auto recharge functionality in Payment service enabled",
     )
 
+    PAYMENTS_FINANCE_DEPARTMENT_EMAIL: EmailStr | None = Field(
+        default=None,
+        description="Special email for finance department. Currently used to BCC invoices.",
```

Comment:
MINOR: Why to I would rather refer to the `BCC email` but not the finance department email. Actually the email you are using is not the finance department. 

In the description is should say that if defined, the payment email notification is BCC to this email

Reply:
Yes you are right, thanks for feedback I will change it now.